src/repositories/sig.py

```
@@ -26,11 +26,21 @@ def get_by_year_semester_not_inactive(
     def get_by_status(self, status: SCSCStatus) -> Sequence[SIG]:
         return self.session.scalars(select(SIG).where(SIG.status == status)).all()
 
-    def get_by_filters(self, filters: dict[str, Any]) -> Sequence[SIG]:
+    def get_by_filters(
+        self,
+        filters: dict[str, Any],
+        tag_texts: list[str] = [],
```

Ruff (0.15.6)

[warning] Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)